google/cloud/bigtable/BUILD

```diff
@@ -38,14 +42,23 @@ cc_library(
     ],
 )
 
+cc_library(
+    name = "bigtable_client",
+    deprecation = "this target will be removed on or around 2022-02-15, please use //:bigtable instead.",
+    tags = ["manual"],
+    deps = [
+        "//google/cloud/bigtable:bigtable_client_internal",
+    ],
+)
+
 load(":bigtable_client_testing.bzl", "bigtable_client_testing_hdrs", "bigtable_client_testing_srcs")
 
 cc_library(
     name = "bigtable_client_testing",
     srcs = bigtable_client_testing_srcs,
     hdrs = bigtable_client_testing_hdrs,
     deps = [
-        ":bigtable_client",
+        "//:bigtable",
```

Comment:
Within this file, using the locally defined `:bigtable_client_internal` seems fine and somewhat more expected.

Reply:
Done.  What is the style for sub-directories (examples, integration tests, benchmarks, etc.)?

Reply:
Not 100% sure, but I think sub-packages would be fully qualified, like `//google/cloud/bigtable/examples:foo`. I can't recall ever seeing relative build targets in BUILD files, other than for targets defined within the local build file.

Reply:
I was going to comment on this, but after seeing that the target also needs to be referred to from `//google/cloud/bigtable` sub-packages (and similarly for the other clients), I favored the simple `//:bigtable` everywhere.

But your choice.

Reply:
FWIW, referring to `//google/cloud/bigtable:bigtable_client_internal` in subpackages would be fine with me, and IMO, preferable to having the bigtable package itself link w/ `//:bigtable`.

But I agree not a big deal. It's not a user-facing issue either way, and we can change this in a separate PR.

Reply:
> FWIW, referring to //google/cloud/bigtable:bigtable_client_internal in subpackages would be fine with me, and IMO, preferable to having the bigtable package itself link w/ //:bigtable.
> 
> But I agree not a big deal. It's not a user-facing issue either way, and we can change this in a separate PR.

Just for the record, the merged version does not appear to use the "internal" target from the subpackages.